.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - '8'
+  - '10'
```

Comment:
Are you sure you want to up this to version 10  given lambda is version 8?

Reply:
Yeah I didn't want to for that very reason. It adds complexity to the whole system. I did it because the newly added `ipfs` module forces it and given the speed at which web3 and ipfs are moving right now, staying deeply backleveled seemed like an issue. Because of their tree's it was also going to be fairly complicated to keep it backleveled too. 

One of the original reasons for webpacking our lambdas was to actually use `babel-env` to allow for async/await in node 6 (previously the only node version supported by lambdas). (As in, writing code in node8, compiling down to node 6). 

AWS doesn't support node10 natively. There are [custom ones](https://github.com/lambci/node-custom-lambda) that do, but it adds ~270ms of overhead which is bad for this application. 

Everything should work without issue even though the runtimes differ. As it turns out though, the lambda doesn't use the ipfs code (at least for now anyway)

The tradeoff of a little more debugging complexity here seemed worth it to me vs the maintenance of old packages.